content/docs/start/data-pipelines.md

```diff
@@ -292,17 +292,16 @@ prepare:
 DVC pipelines (`dvc.yaml` file, `dvc run`, and `dvc repro` commands) solve a few
 important problems:
 
-- _Automation_ - run sequence of steps in a "smart" way that makes iterating on
-  the project faster. It automatically determines which parts of a project need
-  to be run, it caches "runs" and results — all to avoid running the same stage
-  again.
-- _Reproducibility_ - it can describe and capture what data should be used and
-  what commands to run to produce an ML model, for example. It's described and
-  captured in way that is easy to put into Git. It means that it's easy to
-  version and share.
-- _Continuous Delivery and Continuous Integration (CI/CD) for ML_ - describing
-  project in way that it can be reproduced (built) is the fist necessary step
-  before introducing CI/CD systems.
+- _Automation_ - run a sequence of steps in a "smart" way to make iterating on
+  your project faster. DVC caches "runs" and results in stages and automatically
+  determines which parts of a project need to be run to avoid unnecessary
+  re-runs.
+- _Reproducibility_ - `dvc.yaml` and `dvc.lock` files describe and capture what
+  data to use and what commands to run to produce an ML model. Storing these
```

Comment:
> to produce an ML model

It used to be "model, for example". That's because data pipelines can have any results, it doesn't even have to be an ML application. Would be ideal to have it more general (but still mention the "ML model" key word).

Reply:
Maybe something like

...files describe what data to use and what commands to execute to generate results (such as ML models).

Reply:
ah yes I am definitely tunnel visioned toward my use case. Pipeline results is nice and general 👍

Reply:
OK, I'll update it.